### ISSUE TITLE
The tree makes a doctum.js file that is more than 10M in some cases

### ISSUE THREAD

**Issue by williamdes** [code-lts/doctum#43]
https://api.rtdm.info/9.3.0/doctum.js

cc @bradjones1 

Sample

```js
var treeHtml = '<ul><li data-name="namespace:" class="opened"><div style="padding-left:0px" class="hd"><span class="icon icon-play"></span><a href="[Global_Namespace].html">[Global Namespace]</a></div><div class="bd"><ul><li data-name="class:Drupal" ><div style="padding-left:26px" class="hd leaf"><a href="Drupal.html">Drupal</a></div></li></ul></div></li><li data-name="namespace:Drupal" class="opened"><div style="padding-left:0px" class="hd"><span class="icon icon-play"></span><a href="Drupal.html">Drupal</a></div><div class="bd"><ul><li data-name="namespace:Drupal_Component" ><div style="padding-left:18px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component.html">Component</a></div><div class="bd"><ul><li data-name="namespace:Drupal_Component_Annotation" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Annotation.html">Annotation</a></div><div class="bd"><ul><li data-name="namespace:Drupal_Component_Annotation_Doctrine" ><div style="padding-left:54px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Annotation/Doctrine.html">Doctrine</a></div><div class="bd"><ul><li data-name="namespace:Drupal_Component_Annotation_Doctrine_Compatibility" ><div style="padding-left:72px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Annotation/Doctrine/Compatibility.html">Compatibility</a></div><div class="bd"><ul><li data-name="namespace:Drupal_Component_Annotation_Doctrine_Compatibility_Php7" ><div style="padding-left:90px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Annotation/Doctrine/Compatibility/Php7.html">Php7</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Annotation_Doctrine_Compatibility_Php7_ReflectionClass" ><div style="padding-left:116px" class="hd leaf"><a href="Drupal/Component/Annotation/Doctrine/Compatibility/Php7/ReflectionClass.html">ReflectionClass</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Annotation_Doctrine_Compatibility_Php8" ><div style="padding-left:90px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Annotation/Doctrine/Compatibility/Php8.html">Php8</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Annotation_Doctrine_Compatibility_Php8_ReflectionClass" ><div style="padding-left:116px" class="hd leaf"><a href="Drupal/Component/Annotation/Doctrine/Compatibility/Php8/ReflectionClass.html">ReflectionClass</a></div></li></ul></div></li><li data-name="class:Drupal_Component_Annotation_Doctrine_Compatibility_ReflectionClass" ><div style="padding-left:98px" class="hd leaf"><a href="Drupal/Component/Annotation/Doctrine/Compatibility/ReflectionClass.html">ReflectionClass</a></div></li></ul></div></li><li data-name="class:Drupal_Component_Annotation_Doctrine_DocParser" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Annotation/Doctrine/DocParser.html">DocParser</a></div></li><li data-name="class:Drupal_Component_Annotation_Doctrine_SimpleAnnotationReader" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Annotation/Doctrine/SimpleAnnotationReader.html">SimpleAnnotationReader</a></div></li><li data-name="class:Drupal_Component_Annotation_Doctrine_StaticReflectionClass" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Annotation/Doctrine/StaticReflectionClass.html">StaticReflectionClass</a></div></li><li data-name="class:Drupal_Component_Annotation_Doctrine_StaticReflectionParser" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Annotation/Doctrine/StaticReflectionParser.html">StaticReflectionParser</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Annotation_Plugin" ><div style="padding-left:54px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Annotation/Plugin.html">Plugin</a></div><div class="bd"><ul><li data-name="namespace:Drupal_Component_Annotation_Plugin_Discovery" ><div style="padding-left:72px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Annotation/Plugin/Discovery.html">Discovery</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Annotation_Plugin_Discovery_AnnotatedClassDiscovery" ><div style="padding-left:98px" class="hd leaf"><a href="Drupal/Component/Annotation/Plugin/Discovery/AnnotatedClassDiscovery.html">AnnotatedClassDiscovery</a></div></li><li data-name="class:Drupal_Component_Annotation_Plugin_Discovery_AnnotationBridgeDecorator" ><div style="padding-left:98px" class="hd leaf"><a href="Drupal/Component/Annotation/Plugin/Discovery/AnnotationBridgeDecorator.html">AnnotationBridgeDecorator</a></div></li></ul></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Annotation_Reflection" ><div style="padding-left:54px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Annotation/Reflection.html">Reflection</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Annotation_Reflection_MockFileFinder" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Annotation/Reflection/MockFileFinder.html">MockFileFinder</a></div></li></ul></div></li><li data-name="class:Drupal_Component_Annotation_AnnotationBase" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Annotation/AnnotationBase.html">AnnotationBase</a></div></li><li data-name="class:Drupal_Component_Annotation_AnnotationInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Annotation/AnnotationInterface.html">AnnotationInterface</a></div></li><li data-name="class:Drupal_Component_Annotation_Plugin" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Annotation/Plugin.html">Plugin</a></div></li><li data-name="class:Drupal_Component_Annotation_PluginID" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Annotation/PluginID.html">PluginID</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Assertion" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Assertion.html">Assertion</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Assertion_Handle" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Assertion/Handle.html">Handle</a></div></li><li data-name="class:Drupal_Component_Assertion_Inspector" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Assertion/Inspector.html">Inspector</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Bridge" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Bridge.html">Bridge</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Bridge_ZfExtensionManagerSfContainer" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Bridge/ZfExtensionManagerSfContainer.html">ZfExtensionManagerSfContainer</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_ClassFinder" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/ClassFinder.html">ClassFinder</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_ClassFinder_ClassFinder" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/ClassFinder/ClassFinder.html">ClassFinder</a></div></li><li data-name="class:Drupal_Component_ClassFinder_ClassFinderInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/ClassFinder/ClassFinderInterface.html">ClassFinderInterface</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Datetime" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Datetime.html">Datetime</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Datetime_DateTimePlus" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Datetime/DateTimePlus.html">DateTimePlus</a></div></li><li data-name="class:Drupal_Component_Datetime_Time" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Datetime/Time.html">Time</a></div></li><li data-name="class:Drupal_Component_Datetime_TimeInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Datetime/TimeInterface.html">TimeInterface</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_DependencyInjection" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/DependencyInjection.html">DependencyInjection</a></div><div class="bd"><ul><li data-name="namespace:Drupal_Component_DependencyInjection_Dumper" ><div style="padding-left:54px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/DependencyInjection/Dumper.html">Dumper</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_DependencyInjection_Dumper_OptimizedPhpArrayDumper" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/DependencyInjection/Dumper/OptimizedPhpArrayDumper.html">OptimizedPhpArrayDumper</a></div></li><li data-name="class:Drupal_Component_DependencyInjection_Dumper_PhpArrayDumper" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/DependencyInjection/Dumper/PhpArrayDumper.html">PhpArrayDumper</a></div></li></ul></div></li><li data-name="class:Drupal_Component_DependencyInjection_Container" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/DependencyInjection/Container.html">Container</a></div></li><li data-name="class:Drupal_Component_DependencyInjection_PhpArrayContainer" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/DependencyInjection/PhpArrayContainer.html">PhpArrayContainer</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Diff" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Diff.html">Diff</a></div><div class="bd"><ul><li data-name="namespace:Drupal_Component_Diff_Engine" ><div style="padding-left:54px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Diff/Engine.html">Engine</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Diff_Engine_DiffEngine" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Diff/Engine/DiffEngine.html">DiffEngine</a></div></li><li data-name="class:Drupal_Component_Diff_Engine_DiffOp" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Diff/Engine/DiffOp.html">DiffOp</a></div></li><li data-name="class:Drupal_Component_Diff_Engine_DiffOpAdd" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Diff/Engine/DiffOpAdd.html">DiffOpAdd</a></div></li><li data-name="class:Drupal_Component_Diff_Engine_DiffOpChange" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Diff/Engine/DiffOpChange.html">DiffOpChange</a></div></li><li data-name="class:Drupal_Component_Diff_Engine_DiffOpCopy" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Diff/Engine/DiffOpCopy.html">DiffOpCopy</a></div></li><li data-name="class:Drupal_Component_Diff_Engine_DiffOpDelete" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Diff/Engine/DiffOpDelete.html">DiffOpDelete</a></div></li><li data-name="class:Drupal_Component_Diff_Engine_HWLDFWordAccumulator" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Diff/Engine/HWLDFWordAccumulator.html">HWLDFWordAccumulator</a></div></li></ul></div></li><li data-name="class:Drupal_Component_Diff_Diff" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Diff/Diff.html">Diff</a></div></li><li data-name="class:Drupal_Component_Diff_DiffFormatter" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Diff/DiffFormatter.html">DiffFormatter</a></div></li><li data-name="class:Drupal_Component_Diff_MappedDiff" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Diff/MappedDiff.html">MappedDiff</a></div></li><li data-name="class:Drupal_Component_Diff_WordLevelDiff" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Diff/WordLevelDiff.html">WordLevelDiff</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Discovery" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Discovery.html">Discovery</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Discovery_DiscoverableInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Discovery/DiscoverableInterface.html">DiscoverableInterface</a></div></li><li data-name="class:Drupal_Component_Discovery_DiscoveryException" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Discovery/DiscoveryException.html">DiscoveryException</a></div></li><li data-name="class:Drupal_Component_Discovery_YamlDirectoryDiscovery" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Discovery/YamlDirectoryDiscovery.html">YamlDirectoryDiscovery</a></div></li><li data-name="class:Drupal_Component_Discovery_YamlDiscovery" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Discovery/YamlDiscovery.html">YamlDiscovery</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_EventDispatcher" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/EventDispatcher.html">EventDispatcher</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_EventDispatcher_ContainerAwareEventDispatcher" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.html">ContainerAwareEventDispatcher</a></div></li><li data-name="class:Drupal_Component_EventDispatcher_Event" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/EventDispatcher/Event.html">Event</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_FileCache" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/FileCache.html">FileCache</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_FileCache_ApcuFileCacheBackend" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/FileCache/ApcuFileCacheBackend.html">ApcuFileCacheBackend</a></div></li><li data-name="class:Drupal_Component_FileCache_FileCache" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/FileCache/FileCache.html">FileCache</a></div></li><li data-name="class:Drupal_Component_FileCache_FileCacheBackendInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/FileCache/FileCacheBackendInterface.html">FileCacheBackendInterface</a></div></li><li data-name="class:Drupal_Component_FileCache_FileCacheFactory" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/FileCache/FileCacheFactory.html">FileCacheFactory</a></div></li><li data-name="class:Drupal_Component_FileCache_FileCacheInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/FileCache/FileCacheInterface.html">FileCacheInterface</a></div></li><li data-name="class:Drupal_Component_FileCache_NullFileCache" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/FileCache/NullFileCache.html">NullFileCache</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_FileSecurity" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/FileSecurity.html">FileSecurity</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_FileSecurity_FileSecurity" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/FileSecurity/FileSecurity.html">FileSecurity</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_FileSystem" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/FileSystem.html">FileSystem</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_FileSystem_FileSystem" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/FileSystem/FileSystem.html">FileSystem</a></div></li><li data-name="class:Drupal_Component_FileSystem_RegexDirectoryIterator" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/FileSystem/RegexDirectoryIterator.html">RegexDirectoryIterator</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_FrontMatter" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/FrontMatter.html">FrontMatter</a></div><div class="bd"><ul><li data-name="namespace:Drupal_Component_FrontMatter_Exception" ><div style="padding-left:54px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/FrontMatter/Exception.html">Exception</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_FrontMatter_Exception_FrontMatterParseException" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/FrontMatter/Exception/FrontMatterParseException.html">FrontMatterParseException</a></div></li></ul></div></li><li data-name="class:Drupal_Component_FrontMatter_FrontMatter" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/FrontMatter/FrontMatter.html">FrontMatter</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Gettext" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Gettext.html">Gettext</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Gettext_PoHeader" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Gettext/PoHeader.html">PoHeader</a></div></li><li data-name="class:Drupal_Component_Gettext_PoItem" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Gettext/PoItem.html">PoItem</a></div></li><li data-name="class:Drupal_Component_Gettext_PoMemoryWriter" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Gettext/PoMemoryWriter.html">PoMemoryWriter</a></div></li><li data-name="class:Drupal_Component_Gettext_PoMetadataInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Gettext/PoMetadataInterface.html">PoMetadataInterface</a></div></li><li data-name="class:Drupal_Component_Gettext_PoReaderInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Gettext/PoReaderInterface.html">PoReaderInterface</a></div></li><li data-name="class:Drupal_Component_Gettext_PoStreamInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Gettext/PoStreamInterface.html">PoStreamInterface</a></div></li><li data-name="class:Drupal_Component_Gettext_PoStreamReader" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Gettext/PoStreamReader.html">PoStreamReader</a></div></li><li data-name="class:Drupal_Component_Gettext_PoStreamWriter" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Gettext/PoStreamWriter.html">PoStreamWriter</a></div></li><li data-name="class:Drupal_Component_Gettext_PoWriterInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Gettext/PoWriterInterface.html">PoWriterInterface</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Graph" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Graph.html">Graph</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Graph_Graph" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/Graph/Graph.html">Graph</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_HttpFoundation" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/HttpFoundation.html">HttpFoundation</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_HttpFoundation_SecuredRedirectResponse" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/HttpFoundation/SecuredRedirectResponse.html">SecuredRedirectResponse</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_PhpStorage" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/PhpStorage.html">PhpStorage</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_PhpStorage_FileReadOnlyStorage" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/PhpStorage/FileReadOnlyStorage.html">FileReadOnlyStorage</a></div></li><li data-name="class:Drupal_Component_PhpStorage_FileStorage" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/PhpStorage/FileStorage.html">FileStorage</a></div></li><li data-name="class:Drupal_Component_PhpStorage_MTimeProtectedFastFileStorage" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/PhpStorage/MTimeProtectedFastFileStorage.html">MTimeProtectedFastFileStorage</a></div></li><li data-name="class:Drupal_Component_PhpStorage_MTimeProtectedFileStorage" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/PhpStorage/MTimeProtectedFileStorage.html">MTimeProtectedFileStorage</a></div></li><li data-name="class:Drupal_Component_PhpStorage_PhpStorageInterface" ><div style="padding-left:62px" class="hd leaf"><a href="Drupal/Component/PhpStorage/PhpStorageInterface.html">PhpStorageInterface</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Plugin" ><div style="padding-left:36px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Plugin.html">Plugin</a></div><div class="bd"><ul><li data-name="namespace:Drupal_Component_Plugin_Context" ><div style="padding-left:54px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Plugin/Context.html">Context</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Plugin_Context_Context" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Plugin/Context/Context.html">Context</a></div></li><li data-name="class:Drupal_Component_Plugin_Context_ContextDefinitionInterface" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Plugin/Context/ContextDefinitionInterface.html">ContextDefinitionInterface</a></div></li><li data-name="class:Drupal_Component_Plugin_Context_ContextInterface" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Plugin/Context/ContextInterface.html">ContextInterface</a></div></li></ul></div></li><li data-name="namespace:Drupal_Component_Plugin_Definition" ><div style="padding-left:54px" class="hd"><span class="icon icon-play"></span><a href="Drupal/Component/Plugin/Definition.html">Definition</a></div><div class="bd"><ul><li data-name="class:Drupal_Component_Plugin_Definition_ContextAwarePluginDefinitionInterface" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Plugin/Definition/ContextAwarePluginDefinitionInterface.html">ContextAwarePluginDefinitionInterface</a></div></li><li data-name="class:Drupal_Component_Plugin_Definition_ContextAwarePluginDefinitionTrait" ><div style="padding-left:80px" class="hd leaf"><a href="Drupal/Component/Plugin/Definition/ContextAwarePluginDefinitionTrait.html">ContextAwarePluginDefinitionTrait</a></div></li><li data-name="class:Drupal_Component_Plugin_Definition_DerivablePluginDefinitionInterface" ><div style="padding-left:80px" class="hd leaf"><a 
```

**Comment by williamdes** [comment]
This is not finished, but my first fix improved this a bit.

Using Drupal: `9.3.0`
Before: `11,10 Mo`
After: `10,41 Mo`

So all the space is eaten by the search index, incredible.

**Comment by williamdes** [comment]
For the record: https://laravel.com/api/5.8/doctum.js is `222,69 Ko (3,08 Mo un-compressed)`

**Comment by williamdes** [comment]
Closed by bc0be8a988e2766a452e9338657ff47de68462d8

**Comment by williamdes** [comment]
> For the record: https://laravel.com/api/5.8/doctum.js is `222,69 Ko (3,08 Mo un-compressed)`
> 
> After: `2,98 Mo`

After: `87.0 kB`
